src/odemis/acq/stream/test/stream_sparc2_polarizer_test.py

```
@@ -0,0 +1,532 @@
+            l.series_start()
+
+        f = sas.acquire()  # calls acquire method in MultiDetectorStream in sync.py
+
+        # sas.raw: array containing as first entry the sem scan image for the scanning positions,
+        # from second to rest are ar images
+        # data: array should contain same images as sas.raw
+
+        # wait until it's over
+        data, exp = f.result(timeout)
+
+        for l in ars.leeches:
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)